### PR TITLE
Side panel and Individual lists feed

### PR DESCRIPTION
**Pull request by kartikjagdale**
Files to review
1. Api Service.
2. Index , Tweets, UserLists Reducer.
3. Tweets and User List Actions.
4. Tweets and user List Effects.

### CODE REVIEW

**Review by chandradot99 — APPROVED** [review]

File: src/app/components/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -1,48 +1,44 @@
...
-    })
-  }
-
   ngOnInit() {
-    this.data.retriveSuggestion().subscribe(response => {
-      this.lists = response;
-    })
+    this.store.dispatch(this.userListActions.getUserLists());
+    this.store.dispatch(this.userActions.loadProfile());
+    // this.api.getUserDetail().subscribe(response => {
```
Comment (ashish173) [inline src/app/components/dashboard/dashboard.component.ts:38]: @kartikjagdale remove this code.

File: src/app/components/feed/feed.component.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  private subscription: Subscription;
+  private userListId: string;
+  tweets: Observable<any>;
+  constructor(private router: Router,
+              private route: ActivatedRoute,
+              private store: Store<AppState>,
+              private api: ApiService,
+              private tweetActions: TweetsActions) { 
+    this.tweets = this.store.select(getTweets);
+    }
```
Comment (ashish173) [inline src/app/components/feed/feed.component.ts:25]: @kartikjagdale  indentation is not correct here

File: src/app/components/feeds/feeds.component.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<p>
+  feeds works!
```
Comment (ashish173) [inline src/app/components/feeds/feeds.component.html:2]: I think this is not complete yet.

File: src/app/components/suggestions/suggestions.component.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+              private suggestionsActions: SuggestionsActions,
+              private store: Store<AppState>) { 
+    this.suggestedList$ = this.store.select(getSuggestedList);
+}
+
+  ngOnInit() {
+    this.store.dispatch(this.suggestionsActions.retriveLists())
+  }
+
+  follow(listId){
```
Comment (ashish173) [inline src/app/components/suggestions/suggestions.component.ts:30]: @kartikjagdale method should have return types and argument types

File: src/app/effects/user-auth.effect.ts [hunk 5]
```diff
@@ -57,7 +57,7 @@ export class UserAuthEffects {
 
     @Effect() storeUserSuccess$ = this.actions$
       .ofType(ActionTypes.STORE_USER_SUCCESS)
-      .map(() => go(['/dashboard']));
+      .map(() => go(['/dashboard/suggestions']));
```
Comment (ashish173) [inline src/app/effects/user-auth.effect.ts:60]: this is cool I didn't know we could use it like this, Thanks 👍

Comment (ashish173) [reply src/app/effects/user-auth.effect.ts:60]: @pkrawat1  @zeus999 please note this way of routing,

Comment (ashish173) [reply src/app/effects/user-auth.effect.ts:60]: @kartikjagdale its routing right ? or is it something else which I am not aware of ?

**Review by ashish173 — APPROVED** [review]

**Pull request merged**